integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestNonTransactionalCarbonTable.scala

```diff
@@ -2537,6 +2572,52 @@ class TestNonTransactionalCarbonTable extends QueryTest with BeforeAndAfterAll {
     FileUtils.deleteDirectory(new File(writerPath))
   }
 
+  test("Test with long string columns with 1 MB pageSize") {
```

Comment:
seems this testcase has nothing to do with current PR

Reply:
This PR is dependency on PR2814, I already mentioned on PR content,  this test is from PR2814